qemu/tests/nested_test.py

```diff
@@ -0,0 +1,100 @@
+import os
+import logging
+import tempfile
+
+from avocado.core import data_dir
+from avocado.utils import process
+from avocado.utils import software_manager
+
+from virttest import error_context
+from virttest import data_dir as virttest_data_dir
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Nested test:
+    1) Boot VM
+    2) Install ansible and related packages
+    2) Generate inventory file with L1 guest IP
+    3) Execute ansible command
+    4) Copy test results from L1 to host results directory
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    setup_bridge_sh = "/root/setup_bridge.sh"
+
+    def get_live_vms(env):
+        live_vms = []
+        for vm in env.get_all_vms():
+            if vm.is_alive():
+                live_vms.append(vm)
+        if not live_vms:
+            test.fail("No live VM.")
+        return live_vms
+
+    def generate_invent_file(env):
+        vms = get_live_vms(env)
+        invent_file = tempfile.mktemp()
```

Comment:
Suggest put the invent file into either `test.tmpdir` or `test.workdir`

Reference:
https://github.com/avocado-framework/avocado-vt/blob/a689fa089369f00070649d556f77f22e964da242/avocado_vt/test.py#L117
https://avocado-framework.readthedocs.io/en/74.0/api/test/avocado.html#avocado.Test.workdir

Reply:
Put invent file in /var/tmp/avocado_xxxxxx/ directory.